packages/manager/modules/pci/src/projects/project/users/download-openrc/download-openrc.html

```diff
@@ -33,10 +33,6 @@
                 data-match="label">
             </oui-select>
         </oui-field>
-
-        <oui-field
-            data-size="xl">
-            <oui-checkbox model="$ctrl.useVersion3">{{:: 'pci_projects_project_users_download-openrc_v3_label' | translate }}</oui-checkbox>
-        </oui-field>
+        <span data-ng-if="$ctrl.hasGlobalRegions" data-translate="pci_projects_project_users_download-openrc_info_global"></span>
```

Comment:
why don't use `getOpenRcApiVersion` function ?

Reply:
Have changed this